bugbug/repository.py

```diff
@@ -1543,6 +1543,57 @@ def trigger_pull() -> None:
     trigger_pull()
 
 
+def parse_diff(diff_output):
+    parsed_diff = {"added_lines": [], "removed_lines": []}
+    current_file = None
+    lines = diff_output.decode("utf-8", errors="replace").split("\n")
+
+    for line in lines:
+        if line.startswith("diff"):
+            current_file = line.split()[-1]
+        elif line.startswith("+") and not line.startswith("+++"):
+            parsed_diff["added_lines"].append(
+                {"line_content": line[1:], "file": current_file}
+            )
+        elif line.startswith("-") and not line.startswith("---"):
+            parsed_diff["removed_lines"].append(
+                {"line_content": line[1:], "file": current_file}
+            )
+
+    return parsed_diff
```

Comment:
You could try using `rs_parsepatch` instead of parsing the patch manually.

Reply:
Why is that?

Reply:
I'm no longer parsing the patch manually, I'm simply getting the diff between the two commits and storing the raw string so the comment is outdated